{"name": "testcase description", "status": "broken", "statusDetails": {"message": "httprunner.exceptions.ValidationFailure: assert text contains 登录失败，请检查您的用户名或密码是否填写正确(str)\t==> fail\ncheck_item: text\ncheck_value: <html><meta charset='utf-8'/><style>body{background:white}</style><script>alert('ç»å½å¤±è´¥ï¼è¯·æ£æ¥æ¨çç¨æ·åæå¯ç æ¯å¦å¡«åæ­£ç¡®ã')\n</script>\n<html><meta charset='utf-8'/><style>body{background:white}</style><script>if(window.parent) window.parent.document.body.click();\n</script>\n(str)\nassert_method: contains\nexpect_value: 登录失败，请检查您的用户名或密码是否填写正确(str)", "trace": "login_logout_test.py:36: in test_start\n    super().test_start(param)\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:455: in test_start\n    return self.run_testcase(\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:357: in run_testcase\n    extract_mapping = self.__run_step(step)\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:297: in __run_step\n    step_data = self.__run_step_testcase(step)\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:246: in __run_step_testcase\n    testcase_cls()\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:384: in run\n    return self.run_testcase(testcase_obj)\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:357: in run_testcase\n    extract_mapping = self.__run_step(step)\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:295: in __run_step\n    step_data = self.__run_step_request(step)\n..\\..\\venv\\lib\\site-packages\\httprunner\\runner.py:208: in __run_step_request\n    resp_obj.validate(\n..\\..\\venv\\lib\\site-packages\\httprunner\\response.py:271: in validate\n    raise ValidationFailure(failures_string)\nE   httprunner.exceptions.ValidationFailure: assert text contains 登录失败，请检查您的用户名或密码是否填写正确(str)\t==> fail\nE   check_item: text\nE   check_value: <html><meta charset='utf-8'/><style>body{background:white}</style><script>alert('ç»å½å¤±è´¥ï¼è¯·æ£æ¥æ¨çç¨æ·åæå¯ç æ¯å¦å¡«åæ­£ç¡®ã')\nE   </script>\nE   <html><meta charset='utf-8'/><style>body{background:white}</style><script>if(window.parent) window.parent.document.body.click();\nE   </script>\nE   (str)\nE   assert_method: contains\nE   expect_value: 登录失败，请检查您的用户名或密码是否填写正确(str)"}, "description": "TestCase ID: bf618e79-27b1-4a00-a2ea-d8c483152de4", "steps": [{"name": "step: login_$case", "status": "broken", "statusDetails": {"message": "httprunner.exceptions.ValidationFailure: assert text contains 登录失败，请检查您的用户名或密码是否填写正确(str)\t==> fail\ncheck_item: text\ncheck_value: <html><meta charset='utf-8'/><style>body{background:white}</style><script>alert('ç»å½å¤±è´¥ï¼è¯·æ£æ¥æ¨çç¨æ·åæå¯ç æ¯å¦å¡«åæ­£ç¡®ã')\n</script>\n<html><meta charset='utf-8'/><style>body{background:white}</style><script>if(window.parent) window.parent.document.body.click();\n</script>\n(str)\nassert_method: contains\nexpect_value: 登录失败，请检查您的用户名或密码是否填写正确(str)\n", "trace": "  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 357, in run_testcase\n    extract_mapping = self.__run_step(step)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 297, in __run_step\n    step_data = self.__run_step_testcase(step)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 246, in __run_step_testcase\n    testcase_cls()\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 384, in run\n    return self.run_testcase(testcase_obj)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 357, in run_testcase\n    extract_mapping = self.__run_step(step)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 295, in __run_step\n    step_data = self.__run_step_request(step)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 208, in __run_step_request\n    resp_obj.validate(\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\response.py\", line 271, in validate\n    raise ValidationFailure(failures_string)\n"}, "steps": [{"name": "step: login_$case", "status": "broken", "statusDetails": {"message": "httprunner.exceptions.ValidationFailure: assert text contains 登录失败，请检查您的用户名或密码是否填写正确(str)\t==> fail\ncheck_item: text\ncheck_value: <html><meta charset='utf-8'/><style>body{background:white}</style><script>alert('ç»å½å¤±è´¥ï¼è¯·æ£æ¥æ¨çç¨æ·åæå¯ç æ¯å¦å¡«åæ­£ç¡®ã')\n</script>\n<html><meta charset='utf-8'/><style>body{background:white}</style><script>if(window.parent) window.parent.document.body.click();\n</script>\n(str)\nassert_method: contains\nexpect_value: 登录失败，请检查您的用户名或密码是否填写正确(str)\n", "trace": "  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 357, in run_testcase\n    extract_mapping = self.__run_step(step)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 295, in __run_step\n    step_data = self.__run_step_request(step)\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\runner.py\", line 208, in __run_step_request\n    resp_obj.validate(\n  File \"d:\\testdev\\workspace\\python\\venv\\lib\\site-packages\\httprunner\\response.py\", line 271, in validate\n    raise ValidationFailure(failures_string)\n"}, "start": 1617910520521, "stop": 1617910520566}], "start": 1617910520520, "stop": 1617910520566}], "attachments": [{"name": "stderr", "source": "75c39678-5a19-4073-a782-5a462219769a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "param", "value": "{'case': '账号不存在', 'account': 'test', 'password': '333333', 'expect': '登录失败，请检查您的用户名或密码是否填写正确'}"}], "start": 1617910520515, "stop": 1617910520568, "uuid": "75dad190-2cf2-45e6-8f2d-679c3fb31da0", "historyId": "12f2552be78347a81596b88016a7a0d9", "testCaseId": "0ff37cc6b093ec5364e073de52118526", "fullName": "login_logout_test.TestCaseLoginLogout#test_start", "labels": [{"name": "suite", "value": "login_logout_test"}, {"name": "subSuite", "value": "TestCaseLoginLogout"}, {"name": "host", "value": "loong"}, {"name": "thread", "value": "1676-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "login_logout_test"}]}